<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.340">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>ASMS 3D 2023 - 3D print descriptions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>
<style>html{ scroll-behavior: smooth; }</style>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./main.html#category-ms" rel="" target="">
 <span class="menu-text">MS</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./main.html#category-educational" rel="" target="">
 <span class="menu-text">Educational</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./main.html#category-lc" rel="" target="">
 <span class="menu-text">LC</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./main.html#category-labware" rel="" target="">
 <span class="menu-text">Labware</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">3D print descriptions</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<div class="cell" data-execution_count="1">
<div class="cell-output cell-output-display">
<p>3D printing, or additive manufacturing, is a fabrication method that creates three-dimensional objects by depositing materials layer-by-layer, based on a digital model. Models can be printed in a wide range of materials, including polymers, metals, and ceramics. The process begins with a 3D CAD model, which is then sliced into thin layers. 3D printers follow these slices, building the object from the bottom up. Advantages of 3D printing include rapid prototyping, customizability, and “unorthodox” geometries difficult to obtain by traditional manufacturing methods.</p>
<p>This exhibit showcases 3D printed mass spectrometer parts, custom laboratory equipment, and instructive teaching aids. The designs with their description are available on this website. <strong>If using any of these designs in your own work, make sure the material is compatible with the conductivity requirements and environment of the application, including temperature, pressure, and exposure to chemicals or solvents. We hold no responsibility for any harm and/or damage caused by the models on this page.</strong></p>
</div>
<section id="category-lc" class="level2 cell-output cell-output-display">
<h2 class="anchored" data-anchor-id="category-lc">Category: LC</h2>
</section>
<div class="cell-output cell-output-display">
<p><a href="./main.html#-">- - analytical-column-storage-and-transport-box</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#-">- - lc-fitting-union-holder</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#I">I - thermofisher-scientific-lcms-tool-kit</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#J">J - levered-nut-knurl-for-viper-fitting</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#K">K - evosep_hamilton_final.stl</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#L">L - oven-door-v5.stl</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#contact">contact - 3d-printed-sorbent</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#contacted">contacted - bottle-holders-for-inverted-drying-of-solvent-bottles</a></p>
</div>
<section id="-" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="-">- - Analytical Column Storage and Transport Box</h3>
</section>
<section id="author-information" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Thermo Fisher Scientific</p>
</div>
<section id="d-structure-information" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This box can be used to store or transport analytical columns to prevent damage.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Recommend printing in PETG, required no assembly and does not require supports if printed in the orientation with the top of the lid on the build surface. Recommend adding foam inside the box for protecting the column, the foam that column is packaged with can be used.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_100_mm_analytical_column_case_with_lid.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="-" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="-">- - LC fitting union holder</h3>
</section>
<section id="author-information-1" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-1">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-1" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-1">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> A holder to help easily tighten ferrules to LC tubing without having to do it in a port.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed in PLA, no assembly or finishing required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_union_holder_v2_v1.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="I" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="I">I - ThermoFisher Scientific LCMS Tool Kit</h3>
</section>
<section id="author-information-2" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-2">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Thermo Fisher Scientific</p>
</div>
<section id="d-structure-information-2" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-2">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> A toolbox for holding common tools for use with ThermoFisher LC and MS instruments. This keeps your tools neat and organised, so you always know where to find them. It holds two hex keys, a fitting torque wrench, two varieties of transfer tube tools, pump seal replacement tool, two viper plugs, a viper union, U3000 needle replacement wrench, several pump piston alignment tools and a syringe seal removal tool.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> This tool kit can be printed in PLA or PETG and does not require supports or assembly. It is two parts, a body and a lid. To print with Thermo colour theme, print the lid in black before changing to red for the last few layers - this also helps the embossed text stand out.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_lc_toolkit_with_lid.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="J" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="J">J - Levered Nut Knurl for Viper Fitting</h3>
</section>
<section id="author-information-3" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-3">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Thermo Fisher Scientific</p>
</div>
<section id="d-structure-information-3" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-3">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> A Viper Fitting Nut with a lever arm to easily grip to tighten fittings easily in tight places. Can also be used as a printable replacement part if original nut knurl is lost.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Recommend printing in PETG for strenght, no finishing or assembly required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_viper_nut_spanner.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="K" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="K">K - Evosep_Hamilton_Final.stl</h3>
</section>
<section id="author-information-4" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-4">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pyxant Labs/ImaBiotech</p>
</div>
<section id="d-structure-information-4" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-4">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Evosep_Hamilton_Final.stl This is an adapter for the Evosep tip container. It allows the tip container to be used in the Hamilton STAR/STARLet and Nimbus platforms. It also allows the tip container to fit into any place that a standard 96-well deep-well container will fit, including the centrifuges. While not only does this allow for us to create automated methods, it also provides for a better extraction process, as the centrifugation process is noticeably better when the tip containers are locked into place rather than in a bed that is slightly too large.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> material: PETG color: black method of printing: FDM finishing: none assembly: none</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="L" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="L">L - Oven Door v5.stl</h3>
</section>
<section id="author-information-5" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-5">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pyxant Labs/ImaBiotech</p>
</div>
<section id="d-structure-information-5" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-5">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Oven Door v5.stl This is a replacement door for the Shimadzu oven chamber. The factory doors can be broken fairly easily, and is problematic because there is a sensor that will keep the column from heating up if the door is not closed. This replacement allows us to put a new door on that is compatible with the built-in sensors. It also has a physical latch that bypasses the built-in latch to ensure that even if that latch is not functioning, the door will remain in-place during use.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> material: PLA color: Grey method of printing: FDM finishing: none assembly: none</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="contact" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="contact">contact - 3D-Printed sorbent</h3>
</section>
<section id="author-information-6" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-6">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> DAYA RAJU ADYE</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> NATIONAL INSTITUTE OF PHARMACEUTICAL EDUCATION AND RESEARCH, GUWAHATI.</p>
</div>
<section id="d-structure-information-6" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-6">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> 3D printing has emerged as a novel technology for advanced analytical sample preparation techniques. 3D printing enables the prototyping of devices at a low cost. SPE and LLE 3D printed devices have been made in the last couple of years cost-effectively. These successful applications of the new sample preparation method provide a new strategy for the bioanalysis of compounds from complex biological matrices. Exploring new extraction techniques is a trend in analytical chemistry and biomedicine. In the present study, we fabricated our own indigenous native filament using PVA along with Eudragit® RSPO, pH-independent polymer, and plasticizer TEC using the HME technique to enable a customized filament followed by 3D printed sorbent for the extraction of small molecules. The extracted samples were further analyzed using LC-MS/MS. We successfully extracted hydrocortisone from human plasma using a 3D printed sorbent and analyzed it using a validated LC-MS/MS method. The extraction efficiency was &gt;80%. New cost effective extraction techniques using 3D sorbents are one of the most revolutionary techniques in bioanalytical sciences. Our developed 3D sorbent is a milestone that can aid in the development of analytical extraction techniques.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Eudragit® RSPO and PVA were weighed and mixed in a ratio of 1:1 (w/w) with the addition of triethyl citrate (TEC) (8.5 %w/w) as a plasticizer and extruded through a co-rotating twin-screw hot-melt extruder (HAAKE MiniLab II Thermo Scientific, UK) at 60 rpm at a temperature of 120 °C. The average diameter of the extruded filament should be 6.0 cm of extruded length and maintain an acceptable diameter of 2.80 (±0.03) mm for further FDM mediated 3DP, which is in agreement with the diameter of the print-core nozzle, i.e., 2.85 mm.</p>
<p>Computer-aided design (CAD) software SOLIDWORKS (SOLIDWORKS-2019, US) was used to design the prototypes of the 3D sorbent and exported as stereolithography (STL) file. The CAD file dimension of the designed 3D sorbent was 15.0 mm length, 5.0 mm width, 1.5 mm internal cut radius (up to the length), 1.0 mm thickness, and 1.62 mm internal arch used to cut through the thickness. A cylindrical-shaped flake was chosen as the design of the 3D printed sorbent, which made it easy to insert inside the Eppendorf tube. The STL file format contains the desired dimensions of the 3D sorbent needed for the 3D printer slicer software (Ultimaker Cura Version 4.2.1, Singapore) to slice the objects and convert them into a G-code file format. The printing parameters were as follows: layer height, 0.1 mm; in-fill density, 100% (to avoid void formation); infill pattern, triangle; printing temp., 140 C; build-plate temperature, 60 C; printing speed, 70 mm/s; fan speed, 70%; print-core nozzle diameter, 0.4 mm; and total printing time,4min/sorbent. The 3D printed sorbent was printed using a 3D printer (Ultimaker 3, Ultimaker, Germany) using the FDM technology.</p>
<p>The dimensions of the designed cylindrical 3D printed sorbents were 15.0 mm length, 5.0 mm width, 1.5 mm internal cut radius (up to the length), 1.0 mm thickness, and 1.62 mm internal arch (used to cut through the thickness). The cylindrical flake-like geometry provides a larger surface area for better adsorption on a smaller volume space; the small volume space makes it easy to insert the sorbent in a regular Eppendorf tube. As mentioned, the prototype 3D sorbent was cylindrical in shape, slightly yellowish in colour, characteristic odour, non-fragile, and rigid in texture. Surface strandlines were visible at the sorbent surface, which may be due to layer-by-layer extrusion printing.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="contacted" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="contacted">contacted - Bottle holders for inverted drying of solvent bottles</h3>
</section>
<section id="author-information-7" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-7">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> James Hunnicutt and Alan Jarmusch</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> National Institute of Environmental Health Sciences (NIEHS), National Institutes of Health (NIH)</p>
</div>
<section id="d-structure-information-7" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-7">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Holder to safety dry LC solvent bottles in an inverted position</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Fused deposition modeling using PETG</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="category-labware" class="level2 cell-output cell-output-display">
<h2 class="anchored" data-anchor-id="category-labware">Category: Labware</h2>
</section>
<div class="cell-output cell-output-display">
<p><a href="./main.html#?">? - lysis-bead-dispenser-for-barocycler-tubes</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#?">? - pocket-pipette-tip-packs</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#M">M - plate-press-stand-(plate-press-stand-v2.stl)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#N">N - tube-holder-for-hamilton</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#O">O - y-cap</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#P">P - ashwood-labs-high-voltage-test-fixture-1</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#Q">Q - spin96—peptide-sample-clean-up-device-for-stagetips</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#T">T - microarray-mold</a></p>
</div>
<section id="lysis-bead-dispenser-for-barocycler-tubes" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="lysis-bead-dispenser-for-barocycler-tubes">? - Lysis Bead Dispenser for Barocycler Tubes{#?}</h3>
</section>
<section id="author-information-8" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-8">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-8" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-8">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> A bead dispenser for use with barocycler tubes and 1 mm lysis beads. Can dispense into anywhere between 1 and 32 tubes simultaneously saving vast amounts of time during sample prep of proteomic samples that use a barocycler.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed in PLA, no assembly or finishing required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_32_bead_dispenser.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="pocket-pipette-tip-packs" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="pocket-pipette-tip-packs">? - Pocket Pipette Tip Packs{#?}</h3>
</section>
<section id="author-information-9" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-9">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Thermo Fisher Scientific</p>
</div>
<section id="d-structure-information-9" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-9">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Small pipette tip racks that can conveniently transported due to their pocket size. This will allow you to move tips between labs if you are required to work in several locations.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Print in PLA, no supports required. You can print the exchangeable tip holders in the colour theme of the tip volume used by the supplying vendor, for example, grey for 10 uL, yellow for 200 uL and blue for 1000 uL. The lid can be attached to the body by placing a M2*8 set nut into the body and using an 8mm M2 screw through the lid. There is also space to insert 2mm diameter x 1mm deep magnets in the lid to help it stay clipped closed but this is not necessary.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_mini_pipette_tip_travel_case.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="M" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="M">M - Plate Press Stand (Plate Press Stand v2.stl)</h3>
</section>
<section id="author-information-10" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-10">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pyxant Labs/Imabiotech</p>
</div>
<section id="d-structure-information-10" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-10">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This is a print created to help users put caps onto their 96-well sample plate. It allows a user to place the plate into a cutout surrounded by guide rails. The user then puts the cap mat on the plate, then uses a rolling pin to firmly press the caps into position. The cutout keeps the plate firmly in place and the rails ensure that the user does not accidentally flip-over their sample plate during the capping process. This makes it faster, safer, and more efficient to cap plates before analysis.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Material: PETG Color: black Method of Printing: FDM Finishing: None Assembly: None</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_plate_press_stand_v3.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="N" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="N">N - Tube Holder for Hamilton</h3>
</section>
<section id="author-information-11" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-11">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pyxant Labs/Imabiotech</p>
</div>
<section id="d-structure-information-11" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-11">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This is a print that we designed for the Hamilton system. It is specifically for the STAR and STARLet liquid handling systems, and allows users to create bulk QC and STD samples using the automated system. It is designed to hold our, 5mL, 15mL, or 20mL containers and is oriented for convenience. This allows advanced methods to be created and used on our automation hardware.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Material: PLA Color: Grey Mehtod of Printing: FDM Finishing: None Assembly: None</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_tube_holder_hamilton_v4.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="O" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="O">O - Y-Cap</h3>
</section>
<section id="author-information-12" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-12">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pyxant Labs/Imabiotech</p>
</div>
<section id="d-structure-information-12" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-12">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This print allows the use of 2 standard glassware bottle caps to be put on a single bottle. This can be useful in many situations when a single reagent is to be used for multiple pumps. It provides convenience and streamlining processes in the lab.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Material: Polypropylene / CPE Method of Printing: FDM Color: Black (Polypropylene) / Pink (CPE) Finishing: None Assembly: None</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_y-cap-branded.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="P" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="P">P - Ashwood Labs High Voltage Test Fixture 1</h3>
</section>
<section id="author-information-13" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-13">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Mark Osgood</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Ashwood Labs</p>
</div>
<section id="d-structure-information-13" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-13">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The Ashwood Labs High Voltage Test Fixture 1 is for performing high voltage leakage and breakdown testing. At one end the 3D printed body provides a cavity for potting a high resistor, cable and connector terminations. The completed assembly is designed to slide as a sleeve around then end of a Tektronix P6015A high voltage probe. It safely covers all HV connections and holds the probe in a stable position.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Body is Printed in Black PLA with FFF (Fused Filament Fabrication). Breakaway supports used where needed. No post finishing. The completed assembly that this 3D printed body is used in has a connector, an HV resistor and HV leads added. The HV lead terminations are then potted.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="Q" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="Q">Q - Spin96 - Peptide Sample Clean-up Device for StageTips</h3>
</section>
<section id="author-information-14" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-14">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Mark Larance</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> The University of Sydney</p>
</div>
<section id="d-structure-information-14" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-14">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> We aimed to design a 96-well device for use in deep-well centrifuges to increase throughput and consistency when processing StageTips for peptide cleanup, while also minimizing the likelihood of cross-contamination. Our device (Spin96) addresses these issues and was designed with two bottom configurations to allow for a quick changeover between washing and elution steps. Several different plastics for the 3D-printing of Spin96 were tested for durability and usability, with high impact polystyrene (HIPS) chosen for its durability and ease of printing. To optimize the Spin96 protocol, an ideal force of 1000 g was determined as the minimum force required to efficiently pass liquid through StageTips with the Spin96. The protocol for processing SDB-RPS StageTips was also optimized to minimize centrifugation time without compromising the method (see Harney et al.&nbsp;2019. J Proteome Res. 18(5):2228-2240. DOI: 10.1021/acs.jproteome.9b00090).</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> The Spin96 components were exported as STL files for slicing and printing with a layer height of 0.14 mm. Sliced models were printed using HIPS Natural Black/Gray filament on a Zortrax M200 Plus. Printing on the Zortrax M200 Plus was performed according to the high-quality HIPS profile with either maximum infill (used for the holders, wash-bottom and bottom components), or low infill (used for the top component). Print settings used were: (extruder temperature 255 °C, build-plate temperature 80 °C, chamber temperature 40 °C, fan speed 20%, layer height 0.14 mm, print speed 3600 mm/min, retraction distance 0.5 mm, retraction speed 1800 mm/min, infill percentage 95%, use support with maximum overhang angle of 45° and use raft with 5 mm offset). Each part is printed separately and assembled after removal of supports (see Harney et al.&nbsp;2019. J Proteome Res. 18(5):2228-2240. DOI: 10.1021/acs.jproteome.9b00090).</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="T" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="T">T - Microarray Mold</h3>
</section>
<section id="author-information-15" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-15">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Kayla Martinez</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> UTSA</p>
</div>
<section id="d-structure-information-15" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-15">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This device was made to serve as a cost-effective gelatin mold to assist in sample preparation of brain homogenate for sulfatide analysis using LC-MS and TIMS-TOF. In addition to having high overall durability and chemical resistance, the mold can withstand temperatures of -80 degrees celsius with no damage to the mold’s structure. The mold’s custom dimensions and overall design make it easy to remove the gelatin without any breakage or tearing.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> The mold was created using polyethylene terephthalate glycol (PTEG) using UTSA Makerspace’s fused filament fabrication 3D printer. The material comes in various colors, but of the ones available I chose orange. No further assembly is required once the mold has been 3D printed, though it is recommended to ensure that it has no bumps or access material on the surface before using it.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_microarray_mold_1.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="category-educational" class="level2 cell-output cell-output-display">
<h2 class="anchored" data-anchor-id="category-educational">Category: Educational</h2>
</section>
<div class="cell-output cell-output-display">
<p><a href="./main.html#R">R - scale-sciex-6600-qtof</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#S">S - rat-pd-(final_rat.stl)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#U">U - neoantigen-presented-by-hla-molecule</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#U?">U? - 3d-lc-ms-run</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#V">V - covid-spike-and-membrane-(surface)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#W">W - giant-flagellum-(surface)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#X">X - nucleosome-(cartoon)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#Y">Y - alf-protein-structure-(cartoon),-gold-plated-brass-screwed-to-a-black-wooden-base-(so-maybe-more-“art”-than-science?)</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#Z">Z - mass-spectrum-becomes-a-protein-molecular-structure</a></p>
</div>
<section id="R" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="R">R - Scale Sciex 6600 QTOF</h3>
</section>
<section id="author-information-16" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-16">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-16" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-16">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This is a scale model of a Sciex 6600 QTOF Mass Spectrometer.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> The body is printed white, while the top and front panel are printed in blue, the source is printed in grey. All components can be glued together.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_6600.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="S" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="S">S - Rat PD (Final_Rat.stl)</h3>
</section>
<section id="author-information-17" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-17">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shane Karnik</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> ImaBiotech/Pyxant Labs</p>
</div>
<section id="d-structure-information-17" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-17">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Educationnel pieces to leverage the use of whole body animal-tissue imaging as a powefull tools for localizing compounds of biological interest with molecular specificity and high resolution. Applications include pharmaceutical absorptions, distribution, metabolism, excretion studies and biomarker discovery.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Material: PLA Color: grey, red, yellow and green method of printing: FDM Finishing: None Assembly: None</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="U" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="U">U - Neoantigen presented by HLA molecule</h3>
</section>
<section id="author-information-18" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-18">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Shelly Kalaora</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Harvard Medical School</p>
</div>
<section id="d-structure-information-18" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-18">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The structure represents a mutated antigen that was identified in cancer patients and presented on the MHC molecule. This peptide induces an immune response in patients and is used today for treating patients. Using mass spectrometry and Immunopeptidomics we purify the peptides presented on MHC to identify tumor antigens, some of them derived from mutations (and called neo-antigens), like this example. These antigens are later used as part of vaccine (as peptides or RNA, like the COVID-19 vaccine) or as cellular immunotherapy (using the T-cells that recognize these antigens), to treat cancer patients that have these mutations.</p>
<p>Link for the file: https://www.dropbox.com/sh/qowxu1yxe62n7u7/AADtbXEimBVTWTrWAm4sS0P6a?dl=0</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Used PDB structure and ChimeraX to create the 3D printable structure</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="u---3d-lc-ms-runu" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="u---3d-lc-ms-runu">U? - 3D LC-MS run{#U?}</h3>
</section>
<section id="author-information-19" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-19">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Robbin Bouwmeester</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-19" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-19">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="V" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="V">V - Covid spike and membrane (surface)</h3>
</section>
<section id="author-information-20" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-20">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Robbin Bouwmeester</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-20" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-20">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="W" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="W">W - Giant flagellum (surface)</h3>
</section>
<section id="author-information-21" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-21">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Robbin Bouwmeester</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-21" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-21">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="X" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="X">X - Nucleosome (cartoon)</h3>
</section>
<section id="author-information-22" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-22">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Robbin Bouwmeester</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-22" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-22">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="Y" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="Y">Y - ALF protein structure (cartoon), gold-plated brass screwed to a black wooden base (so maybe more “art” than science?)</h3>
</section>
<section id="author-information-23" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-23">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Magnus Palmblad</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-23" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-23">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="Z" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="Z">Z - Mass Spectrum becomes a Protein Molecular Structure</h3>
</section>
<section id="author-information-24" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-24">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Luis Schachner</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Genentech, Inc.</p>
</div>
<section id="d-structure-information-24" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-24">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The 3D printed sculpture I wish to display is a stunning representation of the transformation of a mass spectrum into a molecular structure. The sculpture showcases a seamless blend of art and science, bringing together the two disciplines in a visually striking way. As an additional artistic medium, sculpture offers a unique opportunity to explore the three-dimensional nature of scientific data and brings the data to life in a tangible way.</p>
<p>This sculpture has significant utility for science as well. By visually demonstrating the transformation of a mass spectrum into a molecular structure, the sculpture offers a powerful tool for communicating complex scientific concepts. It can help scientists and non-scientists alike to better understand how mass spectrometry works and the insights it can offer into molecular structures. Additionally, the sculpture can be used to demonstrate the potential of 3D printing technology in scientific research and education.</p>
<p>Overall, the 3D printed sculpture represents an exciting opportunity to showcase the intersection of art and science in a way that is both visually captivating and scientifically informative. It has the potential to inspire curiosity and imagination in viewers and to encourage further exploration of the potential of 3D printing technology in scientific research and education.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> 3D-Printed in nylon and painted with a bronze finish. Please note that STL file will be sent separately to ASMS as the file size is too large.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="category-ms" class="level2 cell-output cell-output-display">
<h2 class="anchored" data-anchor-id="category-ms">Category: MS</h2>
</section>
<div class="cell-output cell-output-display">
<p><a href="./main.html#?\U">?/U - Mass Spectrum becomes a Protein Molecular Structure</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#A">A - printable-framework-for-ambient-ionization-on-commercial-bruker-daltonics-instruments</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#B">B - nanoesi-emitter-holder-cartridge</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#C">C - custom-nano-esi-holder-and-high-resolution-faims-holder-enables-ims-on-bruker-amazon-ion-trap</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#D">D - application-of-3d-printed-tools-for-customized-open-port-probe-electrospray-mass-spectrometry</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#E">E - electrokinetic-stacking-cartridge-for-paper-spray-ms</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#F">F - optiflow-source-button-clip</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#F">F - button-clip-for-zenotof-optiflow-source</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#G">G - mass-spec-rail-horse</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#H">H - low-temperature-plasma-probe-for-in-vivo-aims-and-msi</a></p>
</div>
<div class="cell-output cell-output-display">
<p><a href="./main.html#contact">contact - 1µm-resolution-3d-printing-in-mass-spectrometry</a></p>
</div>
<section id="u---nanu" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="u---nanu">?/U - nan{#?/U}</h3>
</section>
<section id="author-information-25" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-25">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> John Bowling</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> nan</p>
</div>
<section id="d-structure-information-25" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-25">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> nan</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="A" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="A">A - Printable framework for ambient ionization on commercial Bruker Daltonics instruments</h3>
</section>
<section id="author-information-26" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-26">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Kevin Zemaitis</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Pacific Northwest National Laboratory</p>
</div>
<section id="d-structure-information-26" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-26">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The printed source is an inexpensive way to produce reproducible geometries for desorption electrospray ionization (DESI) and more recently low-temperature plasma (LTP) with minimal time and monetary investment. While only these DESI having been demonstrated, and LTP integration in progress, the end user can remix and reuse the ambient source for other purposes. Additionally, any expensive commercial positioners can be replaced with printed or machined solutions for spot profiling analysis, but for imaging, an additional investment is needed for an array of motorized linear stages as described in greater detail elsewhere:</p>
<p>Zemaitis KJ, Izydorczak AM, Thompson AC, Wood TD. Streamlined Multimodal DESI and MALDI Mass Spectrometry Imaging on a Singular Dual-Source FT-ICR Mass Spectrometer. Metabolites. 2021;11(4):253.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed by fused deposition modeling (FDM) this source was originally created on a consumer grade printer (Creality Ender 3) in PLA. Later versions with subtle modifications to the original design to adapt different linear positioners are now printed in ABS on a Prusa i3 Mk3. The material, layer height, nozzle width, etc. will change based upon the printer used, and the end user needs to optimize for best print quality.</p>
<p>The model uses brass inserts for non-galling threads that are heat set into the printed parts, this forms strong connections and modelled parts are built around commercial accessories (e.g., Bruker Daltonics online nanospray adapter). While a common accessory for certain spectrometers (e.g., FTICR MS), we acknowledge not every instrument (e.g., FTICR, TOF, IT) was equipped with a legacy online nanospray source, however, this can be replaced by a printed solution. Given the heat produced within the Apollo II source block, careful consideration should be completed for material choice. The source mounts directly to the Apollo II source and uses the existing atmospheric pressure ionization inlet with a capillary extension. The source and assembly are described in greater detail elsewhere:</p>
<p>Zemaitis KJ, Wood TD. Integration of 3D-printing for a desorption electrospray ionization source for mass spectrometry. Review of Scientific Instruments. 2020;91(10):104102.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="B" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="B">B - NanoESI emitter holder cartridge</h3>
</section>
<section id="author-information-27" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-27">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Carter Asef</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Georgia Institute of Technology</p>
</div>
<section id="d-structure-information-27" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-27">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> A nanoESI emitter must be held in close proximity to the mass spectrometer inlet with fine control on position and angle. To achieve this capability, our lab designed and printed a cartridge that would firmly hold the nanoESI emitter while being positioned with a ring stand and x, y, z manual linear stage. High temperature resin was used to allow for compatibility with mass spectrometer inlet temperatures exceeding 200 degrees Celsius.</p>
<p>The cartridge prints in a single piece with minimal supports and post processing.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Formlabs high temp V2 resin on formlabs form 3 SLA 3d printer. Post processing includes a 6-minute wash in isopropanol followed by 120 minutes in a UV curing station held at 80 degrees Celsius.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_needle_holder_with_clip.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="C" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="C">C - Custom nano-ESI holder and High Resolution FAIMS holder enables IMS on Bruker amaZon Ion Trap</h3>
</section>
<section id="author-information-28" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-28">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Hayden Thurman</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Wichita State University</p>
</div>
<section id="d-structure-information-28" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-28">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> 3d printed nESI holder and high resolution FAIMS holder enables IMS on Bruker amaZon. Pictures of full set up in use on front page of my website: www.nexichem.com</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Black Polyamide carbon fiber printed via high temp FDM (300C). High quality surface finish requires no post processing.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_esi_holder_v5.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="D" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="D">D - Application of 3D printed tools for customized open port probe-electrospray mass spectrometry</h3>
</section>
<section id="author-information-29" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-29">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Gerard Hopfgartner</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> University of Geneva</p>
</div>
<section id="d-structure-information-29" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-29">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The performance of the devices were investigated for the analysis of solid and liquid samples with electrospray ionization mass spectrometry (ESI-MS). Direct analysis in less than 1 min and without any sample preparation, enabled detection of pesticides (azoxtystrobin/imazalil) on fruits peel surface and illegal substances (MDMA/MDEA) in home-made pills. Conjunction of OPP in the overspill mode with a customized autosampler, equipped with disposable pipette tips, enables direct quantitative analysis of drugs of abuse in urine and plasma, with minimized carry-over and reduced matrix effect compared to flow injection analysis.</p>
<p>DOI: 10.1016/j.talanta.2020.120894 DOI: 10.1039/D2AN00925K</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Three dimensional printed open port probe (3DP-OPP) and air displacement based liquid handler, were designed and optimized using fused deposition modeling (FDM) and stereolitography (SLA) 3D printing. The performance of the devices were investigated for the analysis of solid and liquid samples with electrospray ionization mass spectrometry (ESI-MS).</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_opp-2mm.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="E" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="E">E - Electrokinetic Stacking Cartridge for Paper Spray MS</h3>
</section>
<section id="author-information-30" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-30">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Magnus Rydberg</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Indiana University Purdue University Indianapolis</p>
</div>
<section id="d-structure-information-30" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-30">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This device was developed to enable electrokinetic (EK) manipulations (stacking, crude separations, and desalting) on the same paper used to perform paper spray MS. EK stacking improved detection limits 2-3 orders of magnitude with broad analyte compatibility. Online separations and desalting enabled analysis of complex samples with minimal deleterious effects from ion suppression. These improvements were realized without offline sample preparation or the complex instrumentation required to couple HPLC or CE online with MS. To demonstrate the practical utility of the cartridge, per- and polyfluoroalkyl substances (PFAS) were detected quantitatively from tap water with sub-ppt detection limits in less than 10 minutes.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> The cartridge is printed with a Prusa Mini+ using black Fiberology polypropylene filament. Ultimaker adhesion sheets are used and the print bed is not heated. The nozzle diameter is 0.25mm and the layer height 0.1mm. After the print is done, supports and brim are removed with tweezers and scalpel and the print sonicated in 50/50 water/methanol for 10 minutes. Copper wires are attached at the rear of the bottom section for electrical connections and platinum wires are inserted into the liquid reservoirs. The spray substrate is inserted into the top section and attached to the bottom section.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_stacking_cartridge.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="F" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="F">F - Optiflow source button clip</h3>
</section>
<section id="author-information-31" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-31">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-31" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-31">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This clip can be used to hold down the source activation button on a Sciex Optiflow source, which allows you to use a column set up that does not require the Sciex column holder, which may not suit all applications.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed in PLA, no assembly or finishing required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_optiflow_source-button_clip_v3.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="F" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="F">F - Button Clip for ZenoTOF OptiFlow Source</h3>
</section>
<section id="author-information-32" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-32">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-32" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-32">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This clip can be used to hold down the source activation button on a Sciex ZenoTOF OptiFlow Source, which allows you to use a column set up that does not require the Sciex column holder, which may not suit all applications.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed in PLA, no assembly or finishing required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_zenotof_optiflow-button-clip_v4.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="G" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="G">G - Mass Spec Rail Horse</h3>
</section>
<section id="author-information-33" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-33">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Steven Williams</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Children’s Medical Research Institute</p>
</div>
<section id="d-structure-information-33" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-33">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> This model is useful when servicing the ion optics rail of a Sciex 6600 QTOF as it prevents the rail from rolling on the bench while working on it.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Printed in PLA, no assembly or finishing required. Two required.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<p><img src="figs\140001919_rail_horse.png" title="Title" class="img-fluid"></p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="H" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="H">H - Low-Temperature Plasma probe for in-vivo AIMS and MSI</h3>
</section>
<section id="author-information-34" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-34">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Robert Winkler</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> CINVESTAV UGA-Langebio Irapuato, Mexico</p>
</div>
<section id="d-structure-information-34" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-34">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> The 3D-Printed Low-Temperature Plasma (LTP) probe enables the ambient ionization of molecules from solids, liquids, and gases for direct mass spectrometry (MS) and ion mobility spectrometry (IMS) [1,2].<br>
Mounting the 3D-LTP probe on a robotic platform permits the high-throughput screening of spirits and essential oils [3,4]. The special probe design with an inner glass tube reduces the typical hazards of LTP probes (arcing, high temperatures, and ozone generation) and allows the imaging of native biological tissues without sample preparation [5-8]. In addition, the real-time monitoring of volatile organic compounds (VOCs) in biological systems, such as studying the biosynthesis of natural products and plant-microbe interactions, is possible [9,10].</p>
<ol type="1">
<li>Martínez-Jarquín, S.; Moreno-Pedraza, A.; Guillén-Alonso, H.; Winkler, R. Template for 3D Printing a Low-Temperature Plasma Probe. Anal. Chem. 2016, 88 (14), 6976-6980. https://doi.org/10.1021/acs.analchem.6b01019.</li>
<li>Martínez-Jarquín, S.; Winkler, R. Low-Temperature Plasma (LTP) Jets for Mass Spectrometry (MS): Ion Processes, Instrumental Set-Ups, and Application Examples. TrAC Trends in Analytical Chemistry 2017, 89, 133-145. https://doi.org/10.1016/j.trac.2017.01.013.</li>
<li>Rosas-Román, I.; Ovando-Vázquez, C.; Moreno-Pedraza, A.; Guillén-Alonso, H.; Winkler, R. Open LabBot and RmsiGUI: Community Development Kit for Sampling Automation and Ambient Imaging. Microchemical Journal 2020, 152, 104343. https://doi.org/10.1016/j.microc.2019.104343.</li>
<li>Martínez-Jarquín, S.; Moreno-Pedraza, A.; Cázarez-García, D.; Winkler, R. Automated Chemical Fingerprinting of Mexican Spirits Derived from Agave (Tequila and Mezcal) Using Direct-Injection Electrospray Ionisation (DIESI) and Low-Temperature Plasma (LTP) Mass Spectrometry. Anal. Methods 2017, 9 (34), 5023-5028. https://doi.org/10.1039/C7AY00793K.</li>
<li>Martínez‐Jarquín, S.; Winkler, R. Design of a Low‐temperature Plasma (LTP) Probe with Adjustable Output Temperature and Variable Beam Diameter for the Direct Detection of Organic Molecules. Rapid Communications in Mass Spectrometry 2013, 27 (5), 629-634.</li>
<li>García-Rojas, N. S.; Moreno-Pedraza, A.; Rosas-Román, I.; Ramírez-Chávez, E.; Molina-Torres, J.; Winkler, R. Mass Spectrometry Imaging of Thin-Layer Chromatography Plates Using Laser Desorption/Low-Temperature Plasma Ionisation. Analyst 2020, 145 (11), 3885-3891. https://doi.org/10.1039/D0AN00446D.</li>
<li>Moreno-Pedraza, A.; Rosas-Román, I.; Garcia-Rojas, N. S.; Guillén-Alonso, H.; Ovando-Vázquez, C.; Díaz-Ramírez, D.; Cuevas-Contreras, J.; Vergara, F.; Marsch-Martínez, N.; Molina-Torres, J.; Winkler, R. Elucidating the Distribution of Plant Metabolites from Native Tissues with Laser Desorption Low-Temperature Plasma Mass Spectrometry Imaging. Anal. Chem. 2019, 91 (4), 2734-2743. https://doi.org/10.1021/acs.analchem.8b04406.</li>
<li>Maldonado-Torres, M.; López-Hernández, J. F.; Jiménez-Sandoval, P.; Winkler, R. “Plug and Play” Assembly of a Low-Temperature Plasma Ionization Mass Spectrometry Imaging (LTP-MSI) System. Journal of proteomics 2014, 102C, 60-65. https://doi.org/10.1016/j.jprot.2014.03.003.</li>
<li>Torres-Ortega, R.; Guillén-Alonso, H.; Alcalde-Vázquez, R.; Ramírez-Chávez, E.; Molina-Torres, J.; Winkler, R. In Vivo Low-Temperature Plasma Ionization Mass Spectrometry (LTP-MS) Reveals Regulation of 6-Pentyl-2H-Pyran-2-One (6-PP) as a Physiological Variable during Plant-Fungal Interaction. Metabolites 2022, 12 (12), 1231. https://doi.org/10.3390/metabo12121231.</li>
<li>Martínez-Jarquín, S.; Herrera-Ubaldo, H.; de Folter, S.; Winkler, R. In Vivo Monitoring of Nicotine Biosynthesis in Tobacco Leaves by Low-Temperature Plasma Mass Spectrometry. Talanta 2018, 185, 324-327. https://doi.org/10.1016/j.talanta.2018.03.071.</li>
</ol>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> MATERIAL: ABS or generic resin.</p>
<p>COLOR: Any.</p>
<p>METHOD OF PRINTING: Fused filament extrusion for ABS or Photopolymerization for resin.</p>
<p>FINISHING: Sanding and polishing with acetone for ABS or Isopropanol wash and UV light curing for resin.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
<section id="contact" class="level3 cell-output cell-output-display">
<h3 class="anchored" data-anchor-id="contact">contact - 1µm resolution 3D printing in mass spectrometry</h3>
</section>
<section id="author-information-35" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="author-information-35">Author information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>Submitted by:</strong> Vilmos Kertesz</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Affiliation:</strong> Oak Ridge National Laboratory</p>
</div>
<section id="d-structure-information-35" class="level4 cell-output cell-output-display">
<h4 class="anchored" data-anchor-id="d-structure-information-35">3D structure information</h4>
</section>
<div class="cell-output cell-output-display">
<p><strong>General description:</strong> Attached is a small Openport Sampling Interface (OPSI). Also sending link to examples, including an ESI ion source that was already printed (in hand), the size is 400 um. Utility: surface sampling and demo/educational purpose. Magnification glass/optical microscopy will be provided at the conference. The main idea here is, that generally people know about FDM or UV resin printing, both provides around 100 um RESOLUTION. NanoScribe allows 1 um resolution. We were able to add structures inside fused silica capillaries as well, that will be brought too.</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>Description of material:</strong> Material: IP photoresin Color: transparent Method of printing: Two-Photon Polymerization</p>
</div>
<div class="cell-output cell-output-display">
<p><strong>License:</strong> CC-BY</p>
</div>
<div class="cell-output cell-output-display">
<hr>
</div>
</div>



<a onclick="window.scrollTo(0, 0); return false;" role="button" id="quarto-back-to-top"><i class="bi bi-arrow-up"></i> Back to top</a></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
          // target, if specified
          link.setAttribute("target", "_blank");
      }
    }
});
</script>
</div> <!-- /content -->



<script src="site_libs/quarto-html/zenscroll-min.js"></script>
</body></html>